lib/trento/support/mix/tasks/dump_scenario.ex

```diff
@@ -5,9 +5,9 @@ defmodule Mix.Tasks.DumpScenario do
   use Mix.Task
   import Trento.Tasks.Helper
 
-  alias Trento.Integration.Discovery
+  alias Trento.Infrastructure.Discovery
```

Comment:
Not this line specifically, but this module shouldn't be `Mix.Support.Tasks.DumpScenario`.
In general, do we omit the `Support` folder namespace?

Reply:
I added the namespace, thx for spotting it